docs/modules/ROOT/pages/app-installation.adoc

```diff
@@ -0,0 +1,416 @@
+= Installing Scripts as Apps
+:idprefix:
+:idseparator: -
+ifndef::env-github[]
+:icons: font
+endif::[]
+ifdef::env-github[]
+:caution-caption: :fire:
+:important-caption: :exclamation:
+:note-caption: :paperclip:
+:tip-caption: :bulb:
+:warning-caption: :warning:
+endif::[]
+
+JBang allows you to install scripts as system commands, making them available from anywhere on your system just like any other command-line tool.
+
+== Overview
+
+The `jbang app` command system lets you:
+
+- Install JBang scripts as system commands
+- Run scripts from anywhere without typing full paths
+- Manage installed applications
+- Share utilities across your team
```

Comment:
This sentence just seems weird to me. We don't pretend to do anything like this.